src/microbe_stage/Microbe.cs

```diff
@@ -1514,9 +1511,42 @@ private void ApplyATPDamage()
     ///   Handles the death of this microbe. This queues this object
     ///   for deletion and handles some pre-death actions.
     /// </summary>
-    private void HandleDeath()
+    private void HandleDeath(float delta)
     {
-        QueueFree();
+        var cellBurstEffectParticles = new Particles();
```

Comment:
I think `new Particles();` might be mistake here? Should it be `= null` instead?

Reply:
I'll just move it inside the if (!deathParticlesSpawned) and assign it there.